app/backend/server/api/v1/madors_router.py

```diff
@@ -3,18 +3,22 @@
 from sqlalchemy.orm import Session
 from fastapi import APIRouter, Query, Depends, Security
 
-from server.auth import get_current_user
+from server import auth
 
 from db import crud
 from db import schemas
 from db.database import get_db
 
 router = APIRouter()
 
-
-@router.get("/me", response_model=schemas.User)
-async def get_current_user(
-    current_user: schemas.User = Security(get_current_user,
+@router.get("/hierarchy", response_model=List[schemas.Hierarchy])
```

Comment:
What about the mador router we discussed?

Reply:
ive just copied my code and will continue this in another branch